spec/unit/crypto/backup.spec.js

```diff
@@ -361,7 +361,7 @@ describe("MegolmBackup", function() {
                     resolve();
                     return Promise.resolve({});
                 };
-                await client.createKeyBackupVersion({
+                client.createKeyBackupVersion({
```

Comment:
I'm impressed the tests still pass even though the function is in fact async

Reply:
This bit triggers the handler installed above which is wrapped in a promise... It's a bit of a monster. 😅